### DISCUSSION TITLE
栏目高亮设置

### DISCUSSION THREAD

**Discussion by glimmeringlight** [xaoxuu/hexo-theme-stellar#146] [Q&A] [answered]
按照文档配置了友链朋友圈，遇到一个问题，当正在浏览友链朋友圈页面时，导航栏处不会高亮(`class=active`)，请问该如何设置。
![image](https://user-images.githubusercontent.com/99467198/199191181-b32d6ebc-6794-4d04-803a-cb0cea644325.png)
我的配置：
`/link/rss/index.md`:
```markdown
---
seo_title: 朋友文章
# robots: noindex, nofollow
menu_id: post
comments: false
post_list: true
sidebar: [welcome, recent]
---

{% timeline type:fcircle api:https://fcircle.felixchen0707.cn/all %}
{% endtimeline %}
```
`_config.stellar.yaml`
```yaml
######## Index ########
post-index: # 近期发布 分类 标签 归档 and ...
  '朋友文章': /link/rss/
```
另外stellar有没有用户群呐

**Comment by xaoxuu** [comment, marked as answer]
发一下地址看看
你就加volantis的群就行了

**Reply by glimmeringlight** [reply]
地址：https://blog.felixchen0707.cn/link/rss/
OKOK

**Reply by xaoxuu** [reply]
你在这个文件中增加一下日志看看输出：
<img width="774" alt="image" src="https://user-images.githubusercontent.com/16400144/199234665-99c83e47-801c-4bae-8cae-1ae97a66eb3d.png">
```js
if (page.path.includes('rss')) {
  console.log('a:', full_url_for(page.path));
  console.log('b:', full_url_for(obj[key]));
  console.log('c:', url_for(page.path));
  console.log('d:', url_for(obj[key]));
}
```

**Reply by glimmeringlight** [reply]
日志输出如下：
```
a: https://blog.felixchen0707.cn/link/rss/index.html
b: https://blog.felixchen0707.cn/link/rss/
c: /link/rss/index.html
d: /link/rss/
```
似乎是因为后面还跟了`index.html`的问题？

**Reply by glimmeringlight** [reply]
噢噢噢噢貌似在`_config.yml`可以关闭这项，我知道问题了。感谢作者！

**Reply by xaoxuu** [reply]
这我需要兼容一下，不然其他人也会遇到问题

**Reply by xaoxuu** [reply]
不太好弄，两种配置排列组合有4种情况，还是得填写路径的时候注意一下了。
```yaml
post-index: # 近期发布 分类 标签 归档 and ...
  '朋友文章': /link/rss/index.html
```

**Reply by glimmeringlight** [reply]
刚才尝试改了一下代码，然后我在本地换四种配置测试了一下发现没出现问题了（？），不知道行不行
